samples/parallel-processing-california-housing-data/.flake8

```
@@ -0,0 +1,3 @@
+[flake8]
+ignore = E501,E741,E402
+exclude = .git, __pycache__, docs, ml_model/utils, ml_model/conf, ml_model/steps
```

What is the reason to exclude ml_model subfolders? Just curious, nothing I would block for.

---

@kuimoani @kenakamu  do you know why ml_model is excluded from linter? I actually copied this setting from yesno example.

---

ah ok, in that sample there are a lot of scripts from external, which we don't want to modify so therefore the exclusion. We can remove it here.
```suggestion
exclude = .git, __pycache__, docs
```